src/components/UI/Card/CartCards/CartCards.jsx

```diff
@@ -1,88 +1,137 @@
 import axios from 'axios';
 import React from 'react'
-import { useCart } from '../../../../context/CartContext';
-import { useWishList } from '../../../../context/WishListContext';
-
+import {useCart} from '../../../../context/CartContext';
+import {useWishList} from '../../../../context/WishListContext';
+import {UpdateCartService} from '../../../../service/CartService';
+import IcTwotoneShoppingCartCheckout from '../../Icons/IcTwotoneShoppingCartCheckout';
+import IcRoundMinus from '../../Icons/IcRoundMinus';
+import IcRoundPlus from '../../Icons/IcRoundPlus';
+import "./CartCards.css"
+import IcRoundWishlist from '../../Icons/IcRoundWishlist';
 function CartCards(props) {
-    const { _id, title ,productImage , author,price,discount,discountedPrice } =props;
-    const {cartState,setCartState}= useCart();
-    const { WishListState, setWishListState } = useWishList();
-    
-    const RemoveItemsFromCartHandler=async(id)=>{
-        try{
+    const {
+        _id,
+        title,
+        productImage,
+        author,
+        price,
+        discount,
+        discountedPrice,
+        quantity
+    } = props;
+    const {cartState, setCartState} = useCart();
+    const {WishListState, setWishListState} = useWishList();
+
+    const RemoveItemsFromCartHandler = async (id) => {
+        try {
             console.log(id);
-            await axios.delete(`/api/user/cart/${id}`,
-                    { headers:{authorization:localStorage.getItem("feetz")}})
-                        .then((res)=>{
-                            console.log(res);
-                            if(res.status===200){
-                                setCartState(res.data.cart);
-                            }
-                        })
-                        .catch((error)=>{
-                            console.log(error.message);
-                        });
-        }
-        catch(err){
-            console.log("error ",err.message);
+            await axios.delete(`/api/user/cart/${id}`, {
+                headers: {
+                    authorization: localStorage.getItem("feetz")
+                }
+            }).then((res) => {
```

Comment:
You don't need to use then catch inside try catch again. Since you're using async await, you can simple use set the retuned response from the await keyword network call to a variable. ***await*** in an async function suspends the execution till the promise is resolved or rejected.

Reply:
yeah i will fix all those double catch